scripts/install_mnemosyne.sh

```diff
@@ -43,7 +43,7 @@ bash $SCRIPTS/install_mongo.sh
 
 mkdir -p /opt
 cd /opt/
-git clone https://github.com/threatstream/mnemosyne.git
+git clone https://github.com/Antelox/mnemosyne.git
```

Comment:
Once we merge the mnemosyne change, this change won't be necessary.  This change needs to be reverted.  since we would merge all of these at the same time.